README.md

```diff
@@ -69,108 +69,116 @@ For guidance on developing for PowerToys, please read the [developer docs](/doc/
 
 ### Video conference mute
 
-- nVidia Broadcast software won't crash anymore
+- Newly added microphones are now updated and tracked by VCM.
 
 ### Settings
 
-- Upgrade notes in OOBE
-- Fix for settings being lost (This was a hotfix for 0.55)
-- UX improvements. Thanks [@niels9001](https://github.com/niels9001)
+- What's new button in the bottom with a new look, with a couple more UI tweaks. Thanks [@niels9001](https://github.com/niels9001)!
+- Fixed a bug causing Settings not to open when a racing condition caused Keyboard Manager settings to not be read correctly.
+- To open settings from the tray icon a double click is needed instead of a single click.
+- Fix for a bug which would cause checking for updates to run indefinitely.
+- When auto-updating, pass a flag to avoid restarting without being prompted.
 
 ### Installer
 
-- Believe we have a fix for the long hated "app.dark.png is missing" error. Thanks to [@robmen](https://github.com/robmen) for having a great blog!
-- Installer will launch PowerToys under appropriate elevation versus Admin only due to UAC prompt for installation (This was a hotfix for 0.55)
-- PowerToys will now start if installed under different user
+- Dependencies installers are now executed with /norestart to avoid unprompted restarts. (This was a hotfix for 0.56). Thanks [@franky920920](https://github.com/franky920920)!
+- Upgraded .net framework dependency to 6.0.3.
+- Installer logs are now saved where they can be collected and sent by the bug report tool.
+- Reverted changes to start with proper elevation and when installed under a different user since those changes ended up causing more issues where PowerToys would start running with the wrong user.
 
 ### Development
 
-- ARM64 - We removed the last .NET Framework dependency.  Thanks [@snickler](https://github.com/snickler) for helping get this across the finish line!
-- .NET 6 upgrade, now on the newest and hottest .NET runtime
-- Code analyzers have been upgraded!  Thanks [@CleanCodeDeveloper](https://github.com/CleanCodeDeveloper)!
-- Symbols are back!
-- Code refactoring, thanks [@CleanCodeDeveloper](https://github.com/CleanCodeDeveloper) and [@cyberrex5](https://github.com/cyberrex5) for helping here!
-- We are now on VS 2022 with the .NET 6 upgrade.
+- OOBE code refactor to have all module information in XAML, like in Settings. Thanks [@niels9001](https://github.com/niels9001)!
+- Every project now has analyzers turned on and warnings fixed. Thanks [@CleanCodeDeveloper](https://github.com/CleanCodeDeveloper)!
+- New patterns added for code spell-checking and stale entries removed. Thanks [@jsoref](https://github.com/jsoref)
+- Additional logging has been added to Fancy Zones and PowerToys Run.
+- A new CI release build will not be triggered if all that was changed was just documentation.
+- Fixed a racing condition causing flaky build errors when building Power Rename.
```

Comment:
```suggestion
- Fixed a racing condition causing flaky build errors when building PowerRename.
```

Reply:
You're right, no spaces. Thanks for the suggestions. I've committed them.